jpa/hibernate5-legacy/pom.xml

```diff
@@ -59,5 +70,11 @@
             <scope>provided</scope>
         </dependency>
 
+      <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-hibernate-cache-commons</artifactId>
```

Comment:
It his some kind of new bridge for caching?

Reply:
Same as above, for applications that reference the 2lc integration classes by name, so they still deploy on WF13/WF14. We don't need this module, but added it for app compat reasons.